### PR TITLE
Document how to use Jaeger

### LINKED ISSUE #116: Document how to use Jaeger
In the README file we need to document how to use Jaeger

see #115

### PR DESCRIPTION
**Pull request by agoncal**
Fix #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,3 +116,28 @@ The OpenShift descriptor will automatically create `Route`s for Prometheus and J
    > For production-ready Jaeger instances, please see the [Jaeger Operator documentation](https://operatorhub.io/operator/jaeger) for how to properly deploy and configure production-ready instances.
    >
    > For production-ready OpenTelemetry Collector instances, please see the [OpenTelemetry Operator documentation](https://operatorhub.io/operator/opentelemetry-operator) for how to properly deploy and configure production-ready instances.
+
+### Jaeger
+
+Telemetry data is exported to a Jaeger instance, which can be analyzed and visualized at `http://localhost:16686`. 
+By now you've performed a few battles, so let's analyze the telemetry data.
+Open the Jaeger UI by going to `http://localhost:16686`.
```
Comment (edeandrea) [inline README.md:124]: This won't necessarily be true unless they are running everything locally. We mention above (in both the docker compose section & the deploying to kubernetes section) that Jaeger is deployed and how to access it.

Maybe we just say "Open the Jaeger UI based on how you are running the system, either through [Docker Compose](#running-locally-via-docker-compose) or by [deploying the monitoring stack to kubernetes](#monitoring).

File: README.md [hunk 2]
```diff
@@ -116,3 +116,28 @@ The OpenShift descriptor will automatically create `Route`s for Prometheus and J
...
+In the Jaeger UI, select `rest-fights` for the Service and `/api/fights/randomfighters` for the Operation, then click **Find Traces**.
+You should see all the traces corresponding to the request of getting new fighters.
+
+![Jaeger Filters](images/jaeger-2-list-traces.png)
+
+Then, select one trace.
+A trace consists of a series of spans.
+Each span is a time interval representing a unit of work.
+Spans can have a parent/child relationship and form a hierarchy.
+You can see that each trace contains 12 total spans:
```
Comment (edeandrea) [inline README.md:139]: Its actually 14 total spans (I know where you got that number from - I added JDBC tracing last week and the place where you got that was written before I made that change :) )

File: README.md [hunk 3]
```diff
@@ -116,3 +116,28 @@ The OpenShift descriptor will automatically create `Route`s for Prometheus and J
...
+six spans in the `rest-fights` application, four spans in the `rest-heroes` application, and two spans in the `rest-villains` application.
```
Comment (edeandrea) [inline README.md:140]: six spans in `rest-fights`, four spans in `rest-heroes`, and four spans in `rest-villains` - for the same reason as my previous comment.

**Review by edeandrea — CHANGES_REQUESTED** [review]
See in-line comments.

Also, we should add a link at the top of the README which links directly to the Jaeger section.

**Comment by agoncal** [comment]
Done

**Review by edeandrea — APPROVED** [review]

**Pull request merged**